dags/probe_scraper.py

```diff
@@ -94,12 +100,19 @@
         startup_timeout_seconds=360,
         image=probe_scraper_image,
         arguments=probe_scraper_args,
-        email=['telemetry-client-dev@mozilla.com', 'aplacitelli@mozilla.com', 'hwoo@mozilla.com'],
+        email=[
+            "telemetry-client-dev@mozilla.com",
+            "aplacitelli@mozilla.com",
+            "hwoo@mozilla.com",
+            "relud@mozilla.com",
+        ],
         env_vars={
             "AWS_ACCESS_KEY_ID": aws_access_key,
             "AWS_SECRET_ACCESS_KEY": aws_secret_key
         },
-        dag=dag)
+        dag=dag,
+        **airflow_gke_prod_kwargs,
```

Comment:
I figure it's safe to move the currently-still-prod `probe_scraper` task while we're at it because it isn't accessing any gcp resources (other than gke). Added benefit is that then the "find logs" instructions are consistent between this and the `probe-scraper-moz-central` task